index.html

```diff
@@ -78,45 +78,107 @@
-                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
-                    <span class="sr-only">Previous</span>
-                </a>
-                <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
-                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
-                    <span class="sr-only">Next</span>
-                </a>
+
+ <!-- About section -->
+<section class="about">
+    <div id="about-1-background">
+        <div id="about-container">
+
+        <div id = "about-img">
+            <img id="image-showcase" src="images/裂頭的鴨子初稿版.png" alt="Event-Image">
+        </div>
+        <div class="card" id="aboutTextbox">
+            <div class="card-body">
+                <!--<div class = "border"></div>-->
+                <h2 id = "about-intro-1" class="lang" data-key="about-intro-1">About this event</h2>
+                <div class = "border"></div>
+                <div class="about-text"><p class = "lang" data-key="about-intro-2">UTADA Innovation Fair 2020 is a computer programming(coding)-based application
+                    design competition. There are three topics of this event: Artificial Intelligence (AI), University
+                    Life, and Post-Epidemic Era. Participants will brainstorm through the provided themes and
+                    create designs and applications within a limited time. Students with creativity and interest in
+                    computer programming will have an opportunity to communicate with elites from various
+                    industries and to showcase their talents.</p></div>
             </div>
+        </div>
+    </div>
+</div>
+<div class="event-theme">
+    <div class="theme-header">
+    <div class = "border-2"></div>
+    <h1 class = "lang" data-key="theme-header">2020 Themes</h1>
+    <div class = "border-3"></div>
+    </div>
+    <div class = "theme-demo">
+        <div class="first-theme">
+            <img id = "ai-icon" src="https://img.icons8.com/cotton/100/000000/artificial-intelligence.png"/>
+            <h2 class="lang" data-key="theme-1-1">Artificial Intelligence</h2>
+ <!--           <p>One important theme for this event is Articial Intelligence. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. <span id = "more-1">It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</span></p>
```

Comment:
Are these comments suppose to be added up later? If not, we don't need them here.

Reply:
If so, we can leave it here. I think you never used expandText()?